test/functional/JIT_Test/src/jit/test/jitserver/JITServerTest.java

```diff
@@ -662,8 +666,15 @@ public void testServerAOTCache() throws IOException, InterruptedException {
 			destroyCache("test_jitscc");
 			Thread.sleep(DESTROY_SCC_WAIT_TIME_MS);
 
-			if(!checkLogFiles("testServerAOTCache.client"+randomLogId+".jitverboselog.out.*", "remote deserialized"))
+			if(!checkLogFiles("testServerAOTCache.client"+randomLogId+".jitverboselog.out.*", "remote deserialized")) {
+				logger.info("No deserialized methods found at the client.");
+				if(!checkLogFiles("testServerAOTCache.client"+randomLogId+".jitverboselog.out.*", "\\+.*remote")) {
```

Comment:
I suggest using a space before and after  "remote" so that we don't find that particle in the name of the method being compiled. `"\\+.* remote "`